nx-X11/programs/Xserver/Xext/Imakefile

```diff
@@ -79,7 +79,7 @@ XF86BIGFOBJS = xf86bigfont.o
 
        SRCS = shape.c $(SHMSRCS) xcmisc.c\
               xtest.c xtest1di.c xtest1dd.c sleepuntil.c \
-	      bigreq.c sync.c $(SCRNSAVSRC) \
+	      bigreq.c sync.c syncsrv.h $(SCRNSAVSRC) \
```

Comment:
I have been unsure about this, but some lines above I found this, which I took as an example:
```
XF86BIGFSRCS = xf86bigfont.c xf86bigfontsrv.h
```